### ISSUE TITLE
[Bug]: Tabler filled icons missing

### ISSUE THREAD

**Issue by BADtochka** [react-icons/react-icons#940]
### Describe the bug

After upgrading from 5.1.0 to 5.2.0, filled tabler icons disappear from lib

### Expected behavior

.

### Minimal sample repository URL

.

### Steps To Reproduce

1. install 5.2.0 version
2. `cat ./node_modules/react-icons/tb/index.d.ts`

### Package version

v5.2.0

### Additional context

5.1.0:
```TS
...
export declare const TbAlignBoxRightTopFilled: IconType;
export declare const TbAlignBoxRightTop: IconType;
export declare const TbAlignBoxTopCenterFilled: IconType;
export declare const TbAlignBoxTopCenter: IconType;
export declare const TbAlignBoxTopLeftFilled: IconType;
export declare const TbAlignBoxTopLeft: IconType;
export declare const TbAlignBoxTopRightFilled: IconType;
export declare const TbAlignBoxTopRight: IconType;
...
```
5.2.0:
```TS
...
export declare const TbAlignBoxRightTop: IconType;
export declare const TbAlignBoxTopCenter: IconType;
export declare const TbAlignBoxTopLeft: IconType;
export declare const TbAlignBoxTopRight: IconType;
...
```

**Comment by wottpal** [comment]
Great, thanks for fixing this. It seems like `5.2.1` has been released on GitHub but not on npm. 🤔 Is there some ci issue here maybe?

**Comment by kamijin-fanta** [comment]
Sorry. It's a public holiday in Japan. I'll publish it manually next week.

**Comment by kamijin-fanta** [comment]
published v5.2.1 to npm registry
